### DISCUSSION TITLE
Adding multiple transitions in PrioritizedReplayBuffer through add().

### DISCUSSION THREAD

**Discussion by pranshu93** [ymd-h/cpprb#15] [Q&A] [answered]
Hey I am using your package as part of my work and must say it is amazing with the amount of speedup as compared to a naive python implementation. I had one doubt, in my Reinforcement Learning paradigm I have multiple transaction which I store by going individually in a loop. This unnecessary adds to the run time. Is there a way to add multiple transactions simultaneously to the Buffer in a vector operations form?

**Comment by ymd-h** [comment, marked as answer]
@pranshu93
Thank you for your positive feedback. I am very happy to hear that.


`ReplayBuffer` (and its sub-classes) can accept multiple transitions at once.

In [example code](https://ymd_h.gitlab.io/cpprb/features/multistep_add/), 10 transitions are added with single method.

If you still have any questions, please ask us freely.
